notebooks/python/lorenz_parallel.py

```diff
@@ -15,11 +15,11 @@
 # Model parameters
 sigma, rho, beta = 10.0, 28.0, 8/3
 U0 = np.array([5, -5, 20])
-tspan = [0., 5.]
+tspan = [0., 10.]
 lorenz_ = lambda t, state: lorenz(t, state, sigma, rho, beta)
 
 # Parareal params
```

Comment:
What is N here ?

Reply:
please update the comment or use a clearer name !

Reply:
N is the number of sub-intervals